components/cam/src/dynamics/se/inidat.F90

```diff
@@ -453,31 +453,32 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
 
     if ( (ideal_phys .or. aqua_planet)) then
        tmp(:,1,:) = 0._r8
-       phis_tmp(:,:) = 0._r8
+       if (fv_nphys > 0) phis_tmp(:,:) = 0._r8
```

Comment:
Why is this only initialized when fv_nphys > 0? Is there harm in initializing it to 0 regardless?

Reply:
it's only allocated when fv_nphys>0. This matches what's done in E3SM. It certainly could always be initialized, but this was one of the things where I was trying to not disturb how the non-physgrid cases operate.